packages/components/src/components/text/text.jsx

```diff
@@ -1,31 +1,39 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
-    function setStyle() {
+const Text = ({ children, size, color, align, weight, line_height, as, className, ...props }) => {
+    const setStyle = () => {
         const style = {
             '--text-size': `var(--text-size-${size || 's'})`,
             '--text-color': `var(--text-${color || 'general'})`,
-            '--text-lh': `var(--text-lh-${lineHeight || (as === 'p' ? 'm' : 'xs')})`,
+            '--text-lh': `var(--text-lh-${line_height || 'm'})`,
             '--text-weight': `var(--text-weight-${weight || 'normal'})`,
         };
         return style;
-    }
-    function setClassName() {
-        const classStyle = ['dc-text', `${align ? `dc-text--${align}` : ''}`, className];
-        return classStyle;
-    }
+    };
```

Comment:
Could we just define `style` variable here and do like `style={style}`? Just like the case with `className`? We remove the function this way.